src/server/bitops_family.cc

```diff
@@ -850,6 +850,8 @@ void BitOpsFamily::Register(CommandRegistry* registry) {
       << CI{"BITOP", CO::WRITE | CO::NO_AUTOJOURNAL, -4, 2, -1, 1, acl::kBitOp}.SetHandler(&BitOp)
       << CI{"GETBIT", CO::READONLY | CO::FAST, 3, 1, 1, 1, acl::kGetBit}.SetHandler(&GetBit)
       << CI{"SETBIT", CO::WRITE | CO::DENYOOM, 4, 1, 1, 1, acl::kSetBit}.SetHandler(&SetBit);
+
+  builder | "BITPOS" | "BITCOUNT" | "BITFIELD" | "BITFIELD_RO" | "BITOP" | "GETBIT" | "SETBIT";
```

Comment:
@dranikpg I Know what your thoughts are here. Probably you would comment that why don't you move this to the Command constructor such that we get the strings naturally. The reason is:

1. It requires passing the builder to the construction of the CI objects. That means that we make the same amount of line changes
2. I think it's better this way because it decouples it. If we push everything in the constructor of the CI it will be painfully to work with all these parameters
3. The only downside is that I might misspelled a command name. I trust my copy pasting ninja skills so I would say that this won't happen. If it happens we will get a bug report and fix this quickly. Either way not a big deal.

Reply:
You guessed correctly. Duplicating all command names yet a third time is hilarious

I'd argue there is a muuuch better approach. The CommandRegistry is not a relict of the past - change it as you like. Lets add the notion of families there. Lets change it to a CommandRegistryBuilder if it becomes too much.

In short, I'd like to keep existing code as simple as possible. It is possible, without changing all the familes and passing the acl stuff around to keep track of families.

Let then your builder access that information from the command registry. Also if you have it already prepared, you don't need a full blown builder for it